x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/details/index.tsx

```diff
@@ -253,6 +254,18 @@ export const RuleDetailsPageComponent: FC<PropsFromRedux> = ({
 
   const { indicesExist, indexPattern } = useWithSource('default', indexToAdd);
 
+  const exceptionLists = useMemo((): ExceptionIdentifiers[] => {
```

Comment:
Ideally, we'd be transforming all of the rule params to camel case when we fetch them in `use_rule` / `use_rules`. This is a temporary workaround.